### PR TITLE
Update secret.md

### PR DESCRIPTION
**Pull request by jiaj12**
fix wrong text from the different  release

### CODE REVIEW

File: content/zh/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -114,20 +114,17 @@ kubectl create secret generic db-user-pass --from-file=./username.txt --from-fil
...
 
-
-特殊字符（例如 `$`,  `\*` 和 `!` ）需要转义。
-如果您使用的密码具有特殊字符，则需要使用 `\\` 字符对其进行转义。 例如，如果您的实际密码是 `S!B\*d$zDsb` ，则应通过以下方式执行命令：
-     kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password=S\\!B\\\\*d\\$zDsb
-您无需从文件中转义密码中的特殊字符（ `--from-file` ）。
+kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password='S!B\*d$zDsb'
+You do not need to escape special characters in passwords from files (--from-file).  -->
 {{< /note >}}
+特殊字符，例如 $，\，* 和 ！ 将由您的shell解释并需要转义。 在大多数常见的 Shell 中，最简单的转义密码方法是用单引号（'）引起来。 例如，如果您的实际密码是 S!B\*d$zDsb，则应通过以下方式执行命令：
```
Comment (xichengliudui) [inline content/zh/docs/concepts/configuration/secret.md]:
```suggestion
特殊字符，例如 $，\，* 和 ！ 将由 shell 解释并需要转义。 在大多数常见的 Shell 中，最简单的转义密码方法是用单引号（'）引起来。 例如，如果您的实际密码是 S!B\*d$zDsb，则应通过以下方式执行命令：
```

File: content/zh/docs/concepts/configuration/secret.md [hunk 2]
```diff
@@ -114,20 +114,17 @@ kubectl create secret generic db-user-pass --from-file=./username.txt --from-fil
...
 
-
-特殊字符（例如 `$`,  `\*` 和 `!` ）需要转义。
-如果您使用的密码具有特殊字符，则需要使用 `\\` 字符对其进行转义。 例如，如果您的实际密码是 `S!B\*d$zDsb` ，则应通过以下方式执行命令：
-     kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password=S\\!B\\\\*d\\$zDsb
-您无需从文件中转义密码中的特殊字符（ `--from-file` ）。
+kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password='S!B\*d$zDsb'
+You do not need to escape special characters in passwords from files (--from-file).  -->
 {{< /note >}}
+特殊字符，例如 $，\，* 和 ！ 将由 shell 解释并需要转义。 在大多数常见的 Shell 中，最简单的转义密码方法是用单引号（'）引起来。 例如，如果您的实际密码是 S!B\*d$zDsb，则应通过以下方式执行命令：
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/secret.md]: please use backticks where appropriate. for example “`$`、`\*` 和 `!`”。

File: content/zh/docs/concepts/configuration/secret.md [hunk 3]
```diff
@@ -114,20 +114,17 @@ kubectl create secret generic db-user-pass --from-file=./username.txt --from-fil
...
+
+kubectl create secret generic dev-db-secret --from-literal=username=devuser --from-literal=password='S!B\*d$zDsb'
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/secret.md]: please learn yourself some markdown syntax. The original text has indentation that makes the above line embedded code. Those indentation should not be removed.

File: content/zh/docs/concepts/configuration/secret.md [hunk 4]
```diff
@@ -114,20 +114,17 @@ kubectl create secret generic db-user-pass --from-file=./username.txt --from-fil
...
+您无需从文件中转义密码中的特殊字符（--from-file）。
```
Comment (tengqm) [inline content/zh/docs/concepts/configuration/secret.md]: quote "--from-file" with backticks please.

**Pull request closed without merging**